assets/prototype/domain/eventEditor/data/queries/priceTypes/test/usePriceTypes.test.ts

```diff
@@ -0,0 +1,35 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import usePriceTypes from '../usePriceTypes';
+import { ApolloMockedProvider } from '../../../../context';
+import { nodes } from './data';
+import useInitPriceTypeTestCache from './useInitPriceTypeTestCache';
+
+describe('usePriceTypes()', () => {
```

Comment:
I love that this test file is so short and focused.
I feel guilty of having at least one test file that is too long, will need to split that somehow.